src/payment/PaymentForm.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { reduxForm } from 'redux-form';
+import { FormattedMessage } from '@edx/frontend-i18n';
+
+import CardHolderInformation from './CardHolderInformation';
+
+class PaymentForm extends React.Component {
+  onSubmit(values) {
+    /* istanbul ignore next */
```

Comment:
istanbul?

Reply:
Found this: https://disjoint.ca/til/2016/10/30/excluding-code-blocks-from-jest-coverage-reports/

This is temporary until we actually implement this function for realz.